CHANGELOG.md

```diff
@@ -2,12 +2,28 @@
 
 #### 4.2-SNAPSHOT
 
+  Bugs
+
+  Improvements
+
+  Dependency Upgrade
+
+  New Feature
+
+#### 4.2.2
```

Comment:
I recommend adding the date after each release. I found this really helpful when looking at the changelog in retrospective.